chapters/theoretical_foundations/sections/models/subsections/model_animation.tex

```diff
@@ -0,0 +1,75 @@
+\subsection{Model Animation} \label{sec:theory_theory_models_animation}
+The model animations make use of the model armature (skeleton) which can be seen in \autoref{fig:armature}.
+The armature is a set of bones that have different relations to each other and to the mesh.
+The bones create a node structure with the root being the torso bone.
+Most other bones are children of the torso bone or children of children of the torso bone.
+This relation allows the bones to inherit transformations from their parents.
+If a leg bone moves the foot bone will follow. 
+
+It is worth noting that bones do now have to resemble human bones.
+For example in \autoref{fig:armature_foot} we can see a set of bones that are outside of the body.
+These bones are not used to deform the mesh but to interact with other bones.
+Among other things these guarantee that the knee does not bend in the wrong direction.
+
+
+\begin{figure}[H]
+    \centering
+    \begin{subfigure}{0.45\textwidth}
+        \centering
+        \includegraphics[width=0.8\textwidth]{chapters/theoretical_foundations/sections/models/resources/Armature.png}
+        \caption{Armature.}
+        \label{fig:armature}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}{0.45\textwidth}
+        \centering
+        \includegraphics[width=0.8\textwidth]{chapters/theoretical_foundations/sections/models/resources/ArmatureFoot.png}
+        \caption{Abnormal bones.}
+        \label{fig:armature_foot}
+    \end{subfigure}
+
+    \caption{Armature.}
+\end{figure}
+
+The bones need to be connected to the mesh.
+Each vertex is assigned a set of bones that influence it with different weights.
+When bones move the vertices move with them according to how much they move and how much a certain bone influences a certain vertex.
+Assigning weights to vertices in Blender is shown in \autoref{fig:vertex_weights}.
+As seen in the figure it is done by selecting a bone and then painting the vertices with a certain weight.
+
+\begin{figure}[H]
+    \centering
+    \begin{subfigure}{0.45\textwidth}
+        \centering
+        \includegraphics[width=0.8\textwidth]{chapters/theoretical_foundations/sections/models/resources/WeightPaint.png}
+        \caption{Vertex weights.}
+        \label{fig:vertex_weights}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}{0.45\textwidth}
+        \centering
+        \includegraphics[width=0.8\textwidth]{chapters/theoretical_foundations/sections/models/resources/DopeSheet.png}
+        \caption{Keyframes.}
+        \label{fig:keyframes}
+    \end{subfigure}
+
+    \caption{Vertex weights and keyframes.}
+\end{figure}
+
+To define an animation a set of keyframes is used.
+The process of creating the keyframes can be seen in \autoref{fig:keyframes}.
```

Comment:
can we make this picture bigger? It doesn't necessarily have to be right next to "vertex weights" pic